src/app/components/ATIAnalytics/atiUrl/index.ts

```diff
@@ -479,6 +481,10 @@ export const buildReverbAnalyticsModel = ({
           x16: aggregatedCampaigns,
           x17: categoryName,
           x18: isLocServeCookieSet(),
+          ...(experimentVariant && {
+            mv_test: OPTIMIZELY_CONFIG.flagKey,
```

Comment:
What's the difference between `flagKey` & `ruleKey` in Optimizely Config? (And probably not a question for now, but given that they have the same value, can we 🗑️ one & keep the other? - maybe a question for @HarveyPeachey?)

Reply:
They can be different, but I think we should keep them the same.

`flagKey` is the key for the actual experiment and `ruleKey` is the key for the rule on the experiment. We only tend to have 1 rule, but I believe you can have multiple, so potentially multiple `ruleKey` values could exist.

Reply:
I've just used the `flagKey` value for the `mv_test` field as that is the name of the experiment. More of an identifier, so I _think_ its ok to use here?

Reply:
Thanks for explaining. That makes sense to me. Perhaps a future thing to consider when we're tidying this up would be to use flagKey always, and remove ruleKey? (Or even rename it completely)